tests/task_queue_consumer/task_queue_consumer_test.py

```diff
@@ -66,9 +69,10 @@ def expected_params(public_ip_str, scan_targets, message_id):
 
 @pytest.mark.unit
 @serialise_mocks()
-@resetting_mocks(task_queue_consumer.ecs_client, task_queue_consumer.ssm_client)
+@resetting_mocks(task_queue_consumer.ecs_client, task_queue_consumer.ssm_client.get_parameters)
```

Comment:
We said that you need to change these to:

@resetting_mocks(task_queue_consumer.ecs_client.run_task, task_queue_consumer.ssm_client.get_parameters)